tests/e2e/e2e_test.go

```diff
@@ -276,6 +276,32 @@ var _ = Describe("odoe2e", func() {
 			timeoutValue := getPreferenceValue("Timeout")
 			Expect(timeoutValue).To(BeEmpty())
 		})
+
+		It("should set env variables", func() {
+			runCmdShouldPass("odo config set --env PORT=4000")
+			configValue := runCmdShouldPass("odo config view")
+			Expect(configValue).To(ContainSubstring("PORT"))
+			runCmdShouldPass("odo config set --env SECRET_KEY=R2lyaXNoIFJhbW5hbmkgaXMgdGhlIGJlc3Q=")
+			configValue = runCmdShouldPass("odo config view")
```

Comment:
Do more specific  search. For example use ```Value := getConfigValue(KEY)``` instead of ```configValue = runCmdShouldPass("odo config view")```. Also add test steps to verify https://github.com/openshift/odo/pull/1449#pullrequestreview-219098881

Reply:
done